EIPS/eip-3455.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 3455
+title: SUDO Opcode
+author: William Morriss (@wjmelements), Baptiste Vauthey (@thabaptiser)
+discussions-to: https://ethereum-magicians.org/t/eip-3455-sudo-opcode/5860
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-04-01
+---
+
+## Simple Summary
+A new opcode is introduced to allow calling from an arbitrary sender address.
+
+## Abstract
+A new opcode, `SUDO`, is introduced with the same parameters as `CALL`, plus another parameter to specify the sender address.
+
+## Motivation
+There are many use cases for being able to set the sender.
+
+Many tokens are stuck irretrievably because nobody has the key for the owner address.
+In particular, at address zero there is approximately 17 billion USD in tokens and ether according to [etherscan](https://etherscan.io/address/0x0000000000000000000000000000000000000000).
```

Comment:
Hmm, it looks like there is only 16.5 million USD, which unfortunately invalidates the next sentence.

Reply:
It's up to 17.5 *b*illion today.

Reply:
It used to show a number with over 21 zeroes which would have been a better story but I dunno what changed since then